frontend/packages/ceph-storage-plugin/src/components/namespace-store/create-ns-page.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { RouteComponentProps } from 'react-router';
+import { Title } from '@patternfly/react-core';
+import { history } from '@console/internal/components/utils/router';
+import { BreadCrumbs, resourcePathFromModel } from '@console/internal/components/utils';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { ListKind } from 'public/module/k8s/types';
+import CreateNamespaceStoreForm from './create-ns';
+import '../common/noobaa-store/create-bs-ns.scss';
+
+const CreateNamespaceStoreFormPage: React.FC<CreateNamespaceStoreFormPageProps> = ({ match }) => {
+  const { t } = useTranslation();
+  const { ns, appName } = match.params;
+  const [csv, csvLoaded, csvErr] = useK8sGet<ListKind<ClusterServiceVersionKind>>(
+    ClusterServiceVersionModel,
+    '',
+    ns,
+  );
+  const clusterServiceVersion = csvLoaded && !csvErr ? csv.items[0] : null;
+  const onCancel = () => history.goBack();
+
+  return (
+    <>
+      <div className="co-create-operand__header">
+        <div className="co-create-operand__header-buttons">
+          <BreadCrumbs
+            breadcrumbs={[
+              {
+                name:
+                  clusterServiceVersion?.spec?.displayName ||
+                  'Openshift Container Storage Operator',
+                path: resourcePathFromModel(ClusterServiceVersionModel, appName, ns),
+              },
+              { name: t('ceph-storage-plugin~Create Namespace Store'), path: match.url },
+            ]}
+          />
+        </div>
+        <div className="nb-bs-ns-page-title">
+          <Title size="2xl" headingLevel="h1" className="nb-bs-ns-page-title__main">
+            {t('ceph-storage-plugin~Create Namespace Store')}
+          </Title>
+          <p className="nb-bs-ns-page-title__info">
+            {t(
+              'ceph-storage-plugin~Represents an underlying storage to be used as read or write target for the data in the namespace buckets.',
```

Comment:
Is namespace bucket is a CRD ?
What re the CRD noobaa will be pushing apart from Namespace Store underlying ?

Reply:
Namespace store is a CRD that is provided in the CSV along Bucket class, backing store.

Reply:
There is also a namespace bucket that is created with namespace store.
My question was pertaining to that.

Reply:
Oh ! Namespace bucket class is not a separate CRD. The  Bucket Class has additional fields to support this